schema/fda_drugs_enum_schema.mcf

```diff
@@ -0,0 +1,2236 @@
+descriptionURL: "https://www.fda.gov/drugs/data-standards-manual-monographs/route-administration"
+
+Node: dcid:AdministrationRouteEndotracheal
+typeOf: dcs:AdministrationRouteEnum
+name: "Endotracheal"
+fdaCode: "401"
+nciConceptCode: "C38208"
+description: "Administration directly into the trachea."
+fdaShortName: "E-TRACHE"
+descriptionURL: "https://www.fda.gov/drugs/data-standards-manual-monographs/route-administration"
+
+Node: dcid:AdministrationRouteEnteral
+typeOf: dcs:AdministrationRouteEnum
+name: "Enteral"
+fdaCode: "313"
+nciConceptCode: "C38209"
+description: "Administration directly into the intestines."
+fdaShortName: "ENTER"
+descriptionURL: "https://www.fda.gov/drugs/data-standards-manual-monographs/route-administration"
+
+Node: dcid:AdministrationRouteEpidural
+typeOf: dcs:AdministrationRouteEnum
+name: "Epidural"
+fdaCode: "9"
+nciConceptCode: "C38210"
+description: "Administration upon or over the dura mater."
+fdaShortName: "EPIDUR"
+descriptionURL: "https://www.fda.gov/drugs/data-standards-manual-monographs/route-administration"
+
+Node: dcid:AdministrationRouteExtra‑Amniotic
+typeOf: dcs:AdministrationRouteEnum
+name: "Extra‑Amniotic"
+fdaCode: "402"
+nciConceptCode: "C38211"
+description: "Administration to the outside of the membrane enveloping the fetus"
+fdaShortName: "X-AMNI"
+descriptionURL: "https://www.fda.gov/drugs/data-standards-manual-monographs/route-administration"
+
+Node: dcid:AdministrationRouteExtracorporeal
+typeOf: dcs:AdministrationRouteEnum
+name: "Extracorporeal"
+fdaCode: "57"
+nciConceptCode: "C38212"
+description: "Administration outside of the body."
+fdaShortName: "X-CORPOR"
+descriptionURL: "https://www.fda.gov/drugs/data-standards-manual-monographs/route-administration"
+
+Node: dcid:AdministrationRouteHemodialysis
+typeOf: dcs:AdministrationRouteEnum
+name: "Hemodialysis"
+fdaCode: "140"
+nciConceptCode: "C38200"
+description: "Administration through hemodialysate fluid."
+fdaShortName: "HEMO"
+descriptionURL: "https://www.fda.gov/drugs/data-standards-manual-monographs/route-administration"
+
+Node: dcid:AdministrationRouteInfiltration
+typeOf: dcs:AdministrationRouteEnum
+name: "Infiltration"
+fdaCode: "361"
+nciConceptCode: "C38215"
+description: "Administration that results in substances passing into tissue spaces or into cells."
+fdaShortName: "INFIL"
+descriptionURL: "https://www.fda.gov/drugs/data-standards-manual-monographs/route-administration"
+
+Node: dcid:AdministrationRouteInterstitial
+typeOf: dcs:AdministrationRouteEnum
+name: "Interstitial"
+fdaCode: "88"
+nciConceptCode: "C38219"
+description: "Administration to or in the interstices of a tissue."
+fdaShortName: "INTERSTIT"
+descriptionURL: "https://www.fda.gov/drugs/data-standards-manual-monographs/route-administration"
+
+Node: dcid:AdministrationRouteIntraAbdominal
+typeOf: dcs:AdministrationRouteEnum
+name: "Intra-Abdominal"
+fdaCode: "56"
+nciConceptCode: "C38220"
+description: "Administration within the abdomen."
```

Comment:
Do we have schema nodes for properties fdaCode, nciConceptCode, fdaShortName ?

Reply:
@clincoln8 I don't see this changes, but I'm ok with you doing this in a different PR.